code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -1044,6 +1041,7 @@ DEFINE_BUFFER_HANDLER(/obj/machinery/turretid)
 	icon_state = "apc"
 	result_path = /obj/machinery/turretid
 	custom_materials = list(/datum/material/iron=MINERAL_MATERIAL_AMOUNT)
+	pixel_shift = 29
```

Comment:
Also power of 2 (knock it down to 28?); however in this case is a wallmount that magically from an APC frame becomes a Turret control panel, may require a new icon for it.

Reply:
> however in this case is a wallmount that magically from an APC frame becomes a Turret control panel, may require a new icon for it.

I dont see what my pr has to do with that.